s3repo/view.py

```diff
@@ -0,0 +1,26 @@
+from flask.views import MethodView
+from flask import render_template, Response
+import os
```

Comment:
https://www.python.org/dev/peps/pep-0008/#imports (Imports should be grouped in the following order:....)

Reply:
Done.